src/implementations/twig/components/media-container/media-container.html.twig

```diff
@@ -1,8 +1,8 @@
 {% apply spaceless %}
 {#
   Parameters:
-  - "alt" (string) (default: ''),
   - "image" (string) (default: ''),
```

Comment:
having both image and picture is quite confusing.
I think we should at least document it properly in the comment (maybe having a separate section for video)